lib/ansible/modules/identity/ipa/ipa_service.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ipa_service
+author: Cédric Parent
+short_description: Manage FreeIPA service
+description:
+- Add and delete an IPA service using IPA API
+options:
+  krbcanonicalname:
+    description:
+    - principal of the service
+    - Can not be changed as it is the unique identifier.
+    required: true
+    aliases: ["name"]
+  force:
+    description:
+    - Force principal name even if host is not in DNS.
+    required: false
+  state:
+    description: State to ensure
+    required: false
+    default: present
+    choices: ["present", "absent"]
+  ipa_port:
```

Comment:
Replace line 37 to 63 with

```
extends_documentation_fragment: ipa.documentation
```

As this parameters are already documented in IPA documentation fragment.